aa_modules/aamod_firstlevel_contrasts.m

```diff
@@ -387,39 +409,52 @@
 for conind = 1:numel(cons)    
     fullCon = cons{conind}';
     selCon = fullCon(SPM.xX.iC);    
-    columnsCon(conind, :) = selCon;    
-    
+    columnsCon(conind, :) = selCon;      
     % Normalize Contrast
     fullCon = fullCon / max(sum(fullCon(fullCon>0)), sum(fullCon(fullCon < 0)));
     % Calculate efficiency
     effic(conind) = trace(fullCon*iXX*fullCon')^-1;    
 end
 
-% get Text size
-h = figure; set(h, 'Position', [1 1 1280 720]); % HD
-ht = text(1,1,nameCons,'FontSize',12,'FontWeight','Bold','interpreter','none');
+% reduce font size as the number of contrasts to display increases
```

Comment:
You may want to dynamically increase the figure height instead. It will lead to a figure larger than one screen (if presented 1:1), but it is more elegant and scalable.

Reply:
You can also check out my take on this in https://github.com/reprostat/reproanalysis/blob/master/modules/statistics/firstlevelcontrasts.m.

Reply:
I only mentioned adjusting the font size in my description, but I also increase the figure height based on the number of contrasts (cf. lines 428-429). In testing, I found just increasing the figure height was insufficient to get a nice display for the rather large range of contrasts we were using (from 1 to more than 50)

It would be easiest to revisit this if we have a specific test case that gives unacceptable results.